tests/test_transforms/test_encoders/test_categorical_transform.py

```diff
@@ -13,23 +13,33 @@
 from etna.transforms.encoders.categorical import OneHotEncoderTransform
 
 
-@pytest.fixture
-def two_df_with_new_values():
-    d = {
+def get_two_df_with_new_values(dtype: str = "int"):
+    dct_1 = {
         "timestamp": list(pd.date_range(start="2021-01-01", end="2021-01-03")) * 2,
         "segment": ["segment_0"] * 3 + ["segment_1"] * 3,
         "regressor_0": [5, 8, 5, 9, 5, 9],
         "target": [1, 2, 3, 4, 5, 6],
     }
-    df1 = TSDataset.to_dataset(pd.DataFrame(d))
-    d = {
+    df_1 = pd.DataFrame(dct_1)
+    df_1["regressor_0"] = df_1["regressor_0"].astype(dtype)
+    df_1 = TSDataset.to_dataset(df_1)
+
+    dct_2 = {
         "timestamp": list(pd.date_range(start="2021-01-01", end="2021-01-03")) * 2,
         "segment": ["segment_0"] * 3 + ["segment_1"] * 3,
         "regressor_0": [5, 8, 9, 5, 0, 0],
         "target": [1, 2, 3, 4, 5, 6],
     }
-    df2 = TSDataset.to_dataset(pd.DataFrame(d))
-    return df1, df2
+    df_2 = pd.DataFrame(dct_2)
+    df_2["regressor_0"] = df_2["regressor_0"].astype(dtype)
+    df_2 = TSDataset.to_dataset(df_2)
+
+    return df_1, df_2
+
+
+@pytest.fixture
+def two_df_with_new_values():
+    return get_two_df_with_new_values()
```

Comment:
I don't get it - we don't use parameters support, why do we not use fixture as is?